com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/LangServerBootstrapTest.java

```diff
@@ -0,0 +1,113 @@
+/*
+ *
+ *  * Copyright (c) 2019 Broadcom.
```

Comment:
would you like to change with 2020? 😆 🎆 🎄

Reply:
I think we will apply this to the whole codebase in a special commit